### DISCUSSION TITLE
Same Location Name at Different Regions

### DISCUSSION THREAD

**Discussion by Grateful-Viv** [code4recovery/12-step-meeting-list#838] [Q&A] [answered]
I asked about this in my other thread, but I fear it got buried.

How are you able to have the same location name for different regions on your demo site? I see there's "online meeting" at San Jose, "online meeting" at Palo Alto, "online meeting" at Santa Clara, etc.

When I try to select a different region for the same named location, it changes all the meetings with that location name to the new region, or if I try to change the region for the same location name, it changes them all. What am I doing wrong?

**Comment by joshreisner** [comment, marked as answer]
we're not using TSML on aasanjose.org. we're managing our meetings with a Google Sheet and the [C4R Sheets service](https://sheets.code4recovery.org/)

**Reply by Grateful-Viv** [reply]
The demo site is using Google Sheet and the Sheets Service?

In other words, it's not possible for me to have multiple regions for the same location name, correct?

**Reply by joshreisner** [reply]
that's correct. i believe TSML requires you to have a one location name per address and vice versa
